README.md

```diff
@@ -141,8 +153,14 @@ SQL database. If you need an example, you can use: people and wishes
 (one-to-many).
 
 Your answer:
-```
-Replace this with your answer
+```ruby
+class people < ActiveRecord::base
+  has_many wishes
+end
+
+class wish < ActiveRecord::base
+  belongs_to person
+end
 ```
```

Comment:
The Relationships are right, but I would re-examine what the question is asking:
i.e. 
```
Schema define what tables should exist in a database, and what columns those tables should have.

- genie
  - primary key
  - name
- wish
  - primary key
  - genie_id
  - contents
```